### PR TITLE
python310Packages.pywick: fix build; python310Packages.qudida: init at 0.0.4; python310Packages.albumentations: init at 1.3.1

### PR DESCRIPTION
**Pull request by natsukium**
###### Description of changes

- python310Packages.qudida: init at 0.0.4
- python310Packages.albumentations: init at 1.3.1
- python310Packages.pywick: fix build

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 240836` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>10 packages built:</summary>
  <ul>
    <li>python310Packages.albumentations</li>
    <li>python310Packages.albumentations.dist</li>
    <li>python310Packages.pywick</li>
    <li>python310Packages.pywick.dist</li>
    <li>python310Packages.qudida</li>
    <li>python310Packages.qudida.dist</li>
    <li>python311Packages.albumentations</li>
    <li>python311Packages.albumentations.dist</li>
    <li>python311Packages.qudida</li>
    <li>python311Packages.qudida.dist</li>
  </ul>
</details>

File: pkgs/development/python-modules/albumentations/default.nix [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  disabledTests = [
+    # this test hangs up
+    "test_transforms"
+  ];
+
+  pythonImportsCheck = [ "albumentations" ];
+
+  meta = with lib; {
+    description = "Fast image augmentation library and easy to use wrapper around other libraries";
+    homepage = "https://pypi.org/project/albumentations/";
```
Comment (bcdarwin) [inline pkgs/development/python-modules/albumentations/default.nix:61]: nitpick: nixpkgs style for `homepage` seems to be to omit the trailing `/`

Comment (bcdarwin) [inline pkgs/development/python-modules/albumentations/default.nix:61]: consider adding `changelog = "https://github.com/albumentations-team/albumentations/releases/tag/{version}";`

File: pkgs/development/python-modules/pywick/default.nix [hunk 2]
```diff
@@ -68,6 +72,5 @@ buildPythonPackage rec {
     homepage = "https://github.com/achaiah/pywick";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ bcdarwin ];
-    broken = true;  # Nixpkgs missing `albumentations` and `prodict`
```
Comment (bcdarwin) [inline pkgs/development/python-modules/pywick/default.nix:71]: `pywick` seems a bit unmaintained.  I don't use it any longer, so we might consider dropping it if you aren't using it either?

**Review by bcdarwin — APPROVED** [review]

File: pkgs/development/python-modules/pywick/default.nix (hunk 2 above)
Comment (natsukium) [reply pkgs/development/python-modules/pywick/default.nix:71]: I thought I could leave it as long as it works, but would you like me to remove it?
I agree with you, as it is unlikely to be maintained in the future.

Comment (bcdarwin) [reply pkgs/development/python-modules/pywick/default.nix:71]: Up to you; if not, I'll probably remove it later.

Comment (natsukium) [reply pkgs/development/python-modules/pywick/default.nix:71]: Thanks, I will remove it.

**Comment by bcdarwin** [comment]
@natsukium might be worth reopening this for albumentations which seems fairly useful and still maintained?

**Pull request closed without merging**